x-pack/plugin/core/template-resources/src/main/resources/monitoring-beats-mb.json

```diff
@@ -463,9 +810,6 @@
                                 "closed": {
                                   "type": "long"
                                 },
-                                "concurrency": {
```

Comment:
this field is used in stack monitoring, see [code](https://github.com/elastic/kibana/blob/b8c1323d5bdc921a6238003bc698bc03750c5320/x-pack/plugins/monitoring/server/lib/metrics/apm/metrics.ts). Will this break UI?

Reply:
in fact, this field is never emitted by apm-server. Not sure why it is there in the first place.

Reply:
added it back to unblock progress and avoid breaking UI